src/utils/get-schema-directive-from-directive-resolver.ts

```diff
@@ -1,5 +1,6 @@
-import { DirectiveResolverFn, SchemaDirectiveVisitor } from '@kamilkisiela/graphql-tools';
+import {  SchemaDirectiveVisitor } from '@kamilkisiela/graphql-tools/dist/schemaVisitor';
 import { GraphQLField, defaultFieldResolver } from 'graphql';
+import { DirectiveResolverFn } from '@kamilkisiela/graphql-tools/dist/Interfaces';
```

Comment:
?

Reply:
Those are something I forgot related to tree-shaking.